scripts/deploy_arm32.sh

```
@@ -0,0 +1,531 @@
+BLUE='\033[0;34m'
+CYAN='\033[0;36m'
+NC='\033[0m' # No Color
+
+# Configuration
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(dirname "$SCRIPT_DIR")"
+DEPLOY_USER="lavalink"
+DEPLOY_PATH="/opt/lavalink-rust"
+SERVICE_NAME="lavalink-rust"
+CONFIG_DIR="$PROJECT_ROOT/deployment/config"
+SYSTEMD_SERVICE_FILE="$PROJECT_ROOT/deployment/lavalink-rust.service"
```

Shellcheck (0.10.0)

[warning] SYSTEMD_SERVICE_FILE appears unused. Verify use (or export if used externally).

(SC2034)